code/game/objects/structures/kitchen_spike.dm

```diff
@@ -33,10 +59,12 @@
 	var/obj/item/stack/rods/used_rods = attacking_item
 	if(used_rods.get_amount() >= MEATSPIKE_IRONROD_REQUIREMENT)
 		used_rods.use(MEATSPIKE_IRONROD_REQUIREMENT)
-		to_chat(user, span_notice("You add spikes to the frame."))
+		balloon_alert(user, "meatspike built.")
```

Comment:
```suggestion
		balloon_alert(user, "meatspike built!")
```

Do this or remove the punctuation entirely. No periods in balloon alerts: https://hackmd.io/@tgstation/SJIle5wYd#Guidelines